### ISSUE TITLE
No image resizing for pretrained models ?

### ISSUE THREAD

**Issue by Shubhamcl** [mpcrlab/TF_Rover#1]
As per the paper, I see that pre-trained models have been used for comparison. But the file ModelTrain.py doesn't show image resizing or preprocessing as per the pre-trained models. Since model pre-traiend models require resolutions of 224x224 and not 130x320.

Is my assumption wrong? Were the pre-trained models pre-trained by you using some different data ?

Thanks!

**Comment by MichaelTeti** [comment]
The pretrained models were trained on images of the same size, but in the
paper we did not use pretrained models. That is a feature we use in other
work.

Michael

**Comment by Shubhamcl** [comment]
So the results achieved are with models trained from scratch ?

**Comment by MichaelTeti** [comment]
Yes, they were trained from scratch on the same track configuration but
under different lighting conditions and with objects placed on the track.
